sotodlib/hwp/hwp.py

```diff
@@ -54,16 +57,41 @@ def extract_hwpss(aman, signal=None, hwp_angle=None,
     Returns
     -------
     hwpss_stats : AxisManager object
-        The extracted HWPSS and its statistics.
+        The extracted HWPSS and its statistics. The statistics include:
+            - coeffs (n_dets x n_modes) coefficients of the model
+                Sum_n coeffs[2*n]*sin(modes[n]*hwp_angle) + coeffs[2*n+1]*cos(modes[n]*hwp_angle)
+                where the sum on n range(len(modes))
+                Note: n_modes is 2*len(modes)
```

Comment:
This docstring needs to parse properly by sphinx.  The indentation is not compliant.  The "-" in the list items should appear aligned with the preceding paragraph; you can leave a blank line if you want.  Either this::
```
hwpss_stats : AxisManager object
    The extracted HWPSS and its statistics. The statistics include:
    - coeffs (n_dets x n_modes) coefficients of the model
      Sum_n coeffs[2*n]*sin(modes[n]*hwp_angle) + coeffs[2*n+1]*cos(modes[n]*hwp_angle)
      where the sum on n range(len(modes)).  Note: n_modes is 2*len(modes)
```
or this, depending on whether you want the math expression to render as indented.
```
hwpss_stats : AxisManager object
    The extracted HWPSS and its statistics. The statistics include:
    - coeffs (n_dets x n_modes) coefficients of the model
        Sum_n coeffs[2*n]*sin(modes[n]*hwp_angle) + coeffs[2*n+1]*cos(modes[n]*hwp_angle)

      where the sum on n range(len(modes)).  Note: n_modes is 2*len(modes)
```

Reply:
Sorry about that. I figured out how to compile the Sphinx docs before pushing. I checked them, so the formatting should render now. I also added a check for finite covariance matrix when using the curve fitting method in the unit test script. Test script ran and passed on simons1.